index.jsx

```diff
@@ -1,15 +1,27 @@
 import React, {PropTypes} from 'react';
 
-function radio(name, selectedValue, onChange) {
+function radio(kwargs) {
+
   return React.createClass({
     render: function() {
+      const name = kwargs.name;
+
+      // We only want to assign these attribute to the <input />
+      // elements if they were specified explicitly.
+      const optional = {};
+      if(kwargs.hasOwnProperty("selectedValue")) {
```

Comment:
Yeah, I'm just being extra cautious.  Here's the output from the node REPL:

```
> var x = {}
undefined
> x.foo = undefined
undefined
> x
{ foo: undefined }
```

This suggests that it's possible to assign the value ```undefined``` to a key.  So, I decided to play it safe and just use ```hasOwnProperty(...)```.